kerastuner/engine/trial.py

```diff
@@ -17,34 +17,39 @@
 from __future__ import division
 from __future__ import print_function
 
-import os
+import hashlib
 import json
+import os
+import random
+import time
 
+from kerastuner.engine import stateful
 from . import metrics_tracking
 from . import hyperparameters as hp_module
-from . import execution as execution_module
 from ..abstractions import display
 from ..abstractions.tensorflow import TENSORFLOW_UTILS as tf_utils
 
 
-class Trial(object):
+class TrialStatus:
+    RUNNING = "RUNNING"
```

Comment:
Style nit: use `'` for consistency with the rest of the file / codebase

Reply:
Done